charts/opennms-operator/values.yaml

```diff
@@ -3,6 +3,7 @@ operator:
   serviceName: opennms-operator
   image: opennms-operator:bleeding
   imagePullPolicy: IfNotPresent
+  imageUpdateFreq: 60
```

Comment:
The code appears to use this as minutes. Can at least a comment be added here so this is more obvious